apps/explorer/src/components/HomeMetrics/OnTheNetwork.tsx

```diff
@@ -2,58 +2,81 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { useGetTotalTransactionBlocks } from '@mysten/core';
-import { Svg3D32, Nft232, Staking32 } from '@mysten/icons';
 
 import { FormattedStatsAmount } from './FormattedStatsAmount';
 
+import { useGetAddressMetrics } from '~/hooks/useGetAddressMetrics';
 import { useGetNetworkMetrics } from '~/hooks/useGetNetworkMetrics';
 import { Card } from '~/ui/Card';
-import { Heading } from '~/ui/Heading';
+import { Text } from '~/ui/Text';
 
 export function OnTheNetwork() {
     const { data: networkMetrics } = useGetNetworkMetrics();
     const { data: transactionCount } = useGetTotalTransactionBlocks();
+    const { data: addressMetrics } = useGetAddressMetrics();
+
     return (
         <Card bg="lightBlue" spacing="lg" height="full">
-            <Heading color="steel-darker" variant="heading4/semibold">
-                On the Network
-            </Heading>
-            <div className="-mb-3 -mr-8 mt-8 flex gap-8 overflow-x-auto pb-3">
-                <div className="flex gap-8 lg:max-xl:gap-12">
-                    <div className="flex flex-shrink-0 gap-1">
-                        <Svg3D32 className="h-8 w-8 font-normal text-steel-dark" />
+            <div className="flex flex-col gap-4 md:flex-row">
+                <div className="mr-5 flex-1 flex-col">
+                    <div className="flex items-center gap-2">
+                        <Text color="steel-darker" variant="caption/semibold">
+                            On the Network
+                        </Text>
+                        <hr className="flex-1 border-gray-45" />
+                    </div>
+                    <div className="mt-5 flex flex-shrink-0 flex-col gap-2">
```

Comment:
![image](https://github.com/MystenLabs/sui/assets/122397493/24c8824a-194a-4ac5-8dde-7c43cde6cdde)


im sure this will get called out - we'll probably just want the other two cards to grow so that they align at the bottom, but might want to check with @mystie711